docs/site/index/survey-tool/managing-users.md

```diff
@@ -86,43 +100,25 @@ Later on, in the vetting phase, you can send messages with the outstanding dispu
 ## To change users' status
 
 1. Click on the magnifying glass in the left column. You zoom in on that user, and have more Actions in the action column.
-2. Click the one you want, and hit \[Do Action]
-3. Normally you don't want to delete users \-\- you want to Lock them. (Deleting removes information about what they've submitted so far.)
+2. Click the one you want, and click \[Do Action]
+3. If a user is inactive, lock their account. (We don't delete accounts since it removes information about what they've submitted so far.)
```

Comment:
```suggestion
3. If a user is inactive, lock their account. (We don't delete accounts since it removes information about what they've submitted so far, and can be helpful for auditing changes in response to bug reports.)
```

Reply:
Not a bad suggestion, but we don't really need to motivate our choices publicly, better to just keep it short IMHO.